ansible/dom0-bootstrap.yml

```diff
@@ -0,0 +1,18 @@
+---
+- include: ansible-version.yml
+
+- hosts: gh # TODO: it eventually should become `all`
+  become: false
+  remote_user: root
+  gather_facts: false
+  pre_tasks:
+    - name: bootstap python
+      raw: if [ ! -x /usr/bin/python ]; then apt-get update && apt-get -y install python-simplejson python-apt; fi
```

Comment:
:) BTW, this is also useful for people that use a separate virtual environment (ansible_python_interpreter).